site/styles/_typography.scss

```diff
@@ -15,9 +15,9 @@
 }
 
 @mixin heading-1-style() {
-  @include typography-style(32px, 0.4px, $font-weight-normal, 8px, var(--body-color));
+  @include typography-style(32px, 0.4px, $font-weight-normal, 8px, var(--body-color), 42px);
   @include media-breakpoint-up(xs) {
-    @include typography-style(45px);
+    @include typography-style(46px, null, null, null, null, 56px);
```

Comment:
IMO you should change all of these mixin args to keyword args so you don't need to do this!

Reply:
Yeah would be better if it's replaced with the keyword args

Reply:
Yeah would be better if it's replaced with the keyword args

Reply:
- changed good suggestion